examples/shared/shared.rb

```diff
@@ -25,7 +25,7 @@ def init
 
     @@options = {
       # setting host as localhost returns an error on Macs, so check env variable first
-      :host => ENV['AEROSPIKE_HOST'] || 'localhost',
+      :host => ENV['AEROSPIKE_HOSTS'] || 'localhost',
```

Comment:
`AEROSPIKE_HOSTS` can be a comma-separate list of one or more seed nodes, including the port number, e.g. from [the docs](https://github.com/aerospike/aerospike-client-ruby/blob/master/docs/aerospike.md#clientnewhosts-policy-connect):

    ENV["AEROSPIKE_HOSTS"] = "10.0.0.1,10.0.0.2"
    client = Client.new

The client will try to connect to the two seed nodes at 10.0.0.1 and 10.0.0.2 using the default port 3000. To use a different port:

    ENV["AEROSPIKE_HOSTS"] = "10.0.0.1:3010"
    client = Client.new

The client will try to connect to the single seed node at 10.0.0.1 using the port 3010.